pyrk/mod_uncert.py

```diff
@@ -0,0 +1,87 @@
+from ur import units
+from material import Material
+from density_model import DensityModel
+
+
+class Mod(Material):
```

Comment:
What's the story on this mod_uncert file and similar files? Its purpose is not clear to me and it seems to be a duplicate of the Mod class (itself a duplicate of graphite.py, I think.)